site2/website-next/pulsar-cpp/releases.json

```diff
@@ -0,0 +1,3 @@
+[
+  "3.0.0"
```

Comment:
This data file can be a duplicate with `site2/website-next/data/release-cpp.js`. I suppose you resolve data from there (as SSOT) and test/filter whether it's `>= 3.0.0`.

Reply:
Good suggestion. I will fix it soon.

Reply:
Addressed.